Move "fingerprinting vector" definition to Infra Standard

zcorpan opened the pull request:
PR to add equivalent text in Infra:
https://github.com/whatwg/infra/pull/115


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***

### :boom: Error: Wattsi server error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Jan 15, 2021, 7:57 AM UTC)_.

<details>
<summary>More</summary>


PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Wattsi Server](https://github.com/domenic/wattsi-server) - Wattsi Server is the web service used to build the WHATWG HTML spec.

:link: [Related URL](https://build.whatwg.org/wattsi)

```
Parsing MDN data...
Parsing...



```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20whatwg/html%232485.)._
</details>

bucklereed commented:
What's happening to the 'Cross-site communication' section? It'll be left under an unsuitable heading, and talks about 'the problems described above' that have vanished.

zcorpan commented:
@bucklereed thanks for pointing that out, I've moved it to the web-messaging section, before "Security", and renamed that heading to "Privacy considerations".

I've also rebased this on current master.

domenic commented:
Done in https://github.com/whatwg/html/pull/4933.
